src/GridFS/Bucket.php

```diff
@@ -648,6 +661,24 @@ public function rename(mixed $id, string $newFilename)
         }
     }
 
+    /**
+     * Renames all the revisions of a file name in the GridFS bucket.
+     *
+     * @param string $filename    Filename
+     * @param string $newFilename New filename
+     *
+     * @throws FileNotFoundException if no file could be selected
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function renameByName(string $filename, string $newFilename): void
+    {
+        $matchedCount = $this->collectionWrapper->updateFilenameForFilename($filename, $newFilename);
+
+        if (! $matchedCount) {
```

Comment:
```suggestion
        if ($matchedCount === 0) {
```

If consistency with `StreamWrapper::rename()` matters. That method also has the following comment, which may be relevant here, too:

> If $count is null, the update is unacknowledged, the operation is considered successful.